### PR TITLE
behnaz/update-withdrawal-deriv-xfaq

### PR DESCRIPTION
**Pull request by behnaz-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/help-centre/deriv-x.js [hunk 1]
```diff
@@ -189,55 +189,60 @@ const AccountInformation = () => (
...
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('How can I deposit funds into my Deriv X real money account?')}
+        </StyledHeader>
+        <Text>
+            <Localize
+                translate_text="To deposit funds into your Deriv X account on Deriv, you’ll need to use the funds in your Deriv account. Go to Cashier > <0>Transfer between accounts</0> and follow the instructions on the screen."
+                components={[
+                    <ExternalLink
+                        href={`${deriv_app_url}/cashier/account-transfer`}
```
Comment (ShuvoHabib) [inline src/pages/help-centre/deriv-x.js:202]: Add Slash ('/') in the end of the url

File: src/pages/help-centre/deriv-x.js [hunk 2]
```diff
@@ -189,55 +189,60 @@ const AccountInformation = () => (
...
+        </StyledText>
+        <StyledText>
+            {localize(
+                'Your funds will be available in your personal account once your payment is processed. You can check processing times on our Payment methods page.',
+            )}
+        </StyledText>
+        <Text mt="1.7rem">
+            <Localize
+                translate_text="After the required processing time of your selected payment method, your funds will be deposited into your personal account. You can check processing times on our <0>Payment methods</0> page."
+                components={[<LocalizedLinkText to="/payment-methods" color="red" key={0} />]}
```
Comment (ShuvoHabib) [inline src/pages/help-centre/deriv-x.js:241]: Add Slash ('/') in the end of the url

**Review by ShuvoHabib — COMMENTED** [review]
Hi @behnaz-fs just a few minor changes
Add Slash ('/') in the end of the urls

File: src/pages/help-centre/deriv-x.js (hunk 1 above)
Comment (behnaz-fs) [reply src/pages/help-centre/deriv-x.js]: done

File: src/pages/help-centre/deriv-x.js (hunk 2 above)
Comment (behnaz-fs) [reply src/pages/help-centre/deriv-x.js]: done

**Review by habib-binary — APPROVED** [review]
~ LGTM

**Review by habib-binary — APPROVED** [review]
LGTM ~

**Review by habib-binary — APPROVED** [review]
LGTM ~

**Comment by sean-binary** [comment]
Closing this as it is no longer relevant.

**Pull request closed without merging**